source/dataman/DataMan.cpp

```diff
@@ -61,7 +61,10 @@ void DataMan::add_stream(json p_jmsg)
         man->init(p_jmsg);
         this->add_next(method, man);
     }
-    add_man_to_path("zfp", method);
+    if (p_jmsg["compress_method"] != nullptr)
```

Comment:
I have tried it again and I saw this:

libc++abi.dylib: terminating with uncaught exception of type std::domain_error: type must be boolean, but is null
Abort trap: 6

But I think I can replace them with if (msg["compress_method"].is_string())